### PR TITLE
Switch to enum for file location

### PR DESCRIPTION
**Pull request by K-Meech**
Uses enum for file location. Pull request on mobie repo too for this.
For https://github.com/mobie/mobie-viewer-fiji/issues/135

### CODE REVIEW

File: src/main/java/de/embl/cba/tables/FileUtils.java [hunk 1]
```diff
@@ -70,14 +71,15 @@ public static String selectPathFromProjectOrFileSystem (String directory, String
 		if ( directory != null )
 		{
 			final GenericDialog gd = new GenericDialog( "Choose source" );
-			gd.addChoice( "Load from", new String[]{ PROJECT, FILE_SYSTEM }, PROJECT );
+			gd.addChoice( "Load from", new String[]{ FileLocation.Project.toString(),
+					FileLocation.File_system.toString() }, FileLocation.Project.toString() );
 			gd.showDialog();
 			if ( gd.wasCanceled() ) return null;
 			fileLocation = gd.getNextChoice();
```
Comment (tischi) [inline src/main/java/de/embl/cba/tables/FileUtils.java:78]: I think the point would be to make it an enum entry immediately:

`fileLocation = FileLocation.valueOf( gd.getNextChoice() );`

File: src/main/java/de/embl/cba/tables/FileUtils.java [hunk 2]
```diff
@@ -70,14 +71,15 @@ public static String selectPathFromProjectOrFileSystem (String directory, String
...
 		}
 
 		String filePath = null;
-		if ( directory != null && fileLocation.equals( PROJECT ) && directory.contains( "raw.githubusercontent" ) )
+		if ( directory != null && fileLocation.equals( FileLocation.Project.toString() ) && directory.contains( "raw.githubusercontent" ) )
```
Comment (tischi) [inline src/main/java/de/embl/cba/tables/FileUtils.java:82]: ...and then here:

`...fileLocation.equals( FileLocation.Project )...`

**Review by tischi — COMMENTED** [review]
small suggestions.

**Comment by K-Meech** [comment]
Fixed! Thanks for pointing out .valueOf - still getting to grips with java enums

**Comment by tischi** [comment]
> still getting to grips with java enums

me too :)

**Pull request merged**